### PR TITLE
update everyone to 23811

### PR DESCRIPTION
**Pull request by anurse**
add a little hackiness to the build script to use 23811 in the stage 0 as well

/cc @piotrpMSFT @Sridhar-MS @brthor @livarcocc

### CODE REVIEW

**Comment by anurse** [comment]
@dotnet-bot test this please

File: test/Installer/Dotnet.Cli.Msi.Tests/project.json [hunk 1]
```diff
@@ -14,4 +14,4 @@
          }
 	  }
   }
-}
\ No newline at end of file
+}
```
Comment (livarcocc) [inline test/Installer/Dotnet.Cli.Msi.Tests/project.json]: This guy doesn't have NetStandard?

**Comment by livarcocc** [comment]
LGTM

File: test/Installer/Dotnet.Cli.Msi.Tests/project.json (hunk 1 above)
Comment (anurse) [inline test/Installer/Dotnet.Cli.Msi.Tests/project.json]: It only runs on Windows and targets `net46`

**Comment by anurse** [comment]
@dotnet-bot test Ubuntu Release Build please

**Comment by anurse** [comment]
@dotnet-bot test OSX Debug Build please

**Comment by anurse** [comment]
@dotnet-bot test OSX Debug Build please

**Pull request merged**